script.js

```diff
@@ -0,0 +1,98 @@
+import {languageDetailsMap} from './details.js';
+
+document.addEventListener("DOMContentLoaded", function() {
+
+    const RGB_BLACK_COLOR = "rgb(0, 0, 0)";
+    const RGB_YELLOW_COLOR = "rgb(255, 255, 0)";
+    const BLUR_EFFECT = "blur(5px)";
```

Comment:
Nie bardzo widzę robić z tego stałą poza funkcją. Używasz tego tylko raz, więc lepiej umieść to w funkcji.
Generalnie to dobra praktyka co zrobiłeś, ale w tym przypadku to zbędne. Z pozostałymi stałymi podobnie. Przenieś je do wnętrza funkcji jak są tylko raz wykorzystane. W tym przypadku to tylko zaciemnia.